driver-core/src/main/com/mongodb/client/model/search/SearchOperator.java

```diff
@@ -322,6 +322,36 @@ static PhraseSearchOperator phrase(final Iterable<? extends SearchPath> paths, f
                 .append("query", queryIterator.hasNext() ? queries : firstQuery));
     }
 
+    /**
+     * Returns a {@link SearchOperator} that performs a search using a special characters in the search string that can match any character.
+     *
+     * @param query The string to search for.
+     * @param path The indexed field to be searched.
+     * @return The requested {@link SearchOperator}.
+     * @mongodb.atlas.manual atlas-search/wildcard/ wildcard operator
+     */
+    static WildcardSearchOperator wildcard(final String query, final SearchPath path) {
+        return wildcard(singleton(notNull("query", query)), singleton(notNull("path", path)));
+    }
+
+    /**
+     * Returns a {@link SearchOperator} that enables queries which use special characters in the search string that can match any character.
```

Comment:
```suggestion
     * Returns a {@link SearchOperator} that performs a search using a special characters in the search string that can match any character.
```
to match the other change